scala/scala-impl/src/org/jetbrains/plugins/scala/codeInspection/scalastyle/ScalastyleCodeInspection.scala

```diff
@@ -53,9 +53,7 @@ object ScalastyleCodeInspection {
       possibleConfigFileNames.flatMap(name => Option(dir.findChild(name))).headOption
 
     def findIn(project: Project, possibleConfigFileNames: Seq[String]): Option[VirtualFile] = {
-      val root = project.getBaseDir
-      if (root == null) return None
-
+      val root = ProjectUtil.guessProjectDir(project)
```

Comment:
Slight drive-by refactor, but I had trouble testing this change with a different plugin and `getBaseDir` is deprecated.

Reply:
This looks good, we'll keep that :+1:
But with the null-check... or was there a specific reason you removed that?

Reply:
Whoops, nope added back :)